spring.beans/src/org/netbeans/modules/spring/beans/resources/templates/springXMLConfig.template

```diff
@@ -1,4 +1,24 @@
 <?xml version="1.0" encoding="${project.encoding}"?>
+<#--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
```

Comment:
Please undo the license header for this .template file and exclude it from nbbuild/build.xml for the "rat" task.

See how it was done for this other PR: https://github.com/emilianbold/incubator-netbeans/pull/282/files#diff-fa67ba03971f6d21c0ff2493bf5d922f

Reply:
Done